packages/core/src/util/atMostOnce.js

```diff
@@ -1,41 +0,0 @@
-/*
```

Comment:
Move the content of the file "atMostOnce" into this file for now, because I could not find a nice way to add an init fn to this little helper.